src/metabase/models/serialization/util.clj

```diff
@@ -345,6 +345,13 @@
 
 (declare ^:private mbql-deps-map)
 
+(def ^:private link-card-model->toucan-model-str
+  {"card"      "Card"
+   "dataset"   "Card"
+   "table"     "Table"
+   "database"  "Database"
+   "dashboard" "Dashboard"})
```

Comment:
missing collection again

Reply:
Should we reuse `link-card-model->toucan-model` and get the string from the model key? Then we wouldn't have to update two places to serialize another model